resonate/resonate.py

```diff
@@ -199,17 +201,24 @@ def remote(
 
     @property
     def promises(self) -> PromiseStore:
+        """Promises low level client."""
         return self._store.promises
 
     @property
     def schedules(self) -> ScheduleStore:
+        """Schedules low level client."""
         return self._store.schedules
 
     def start(self) -> None:
+        """Start resonate.
+
+        This is automatically executed when calling `.run()` or `.rpc()`
+        """
```

Comment:
Maybe something like:

"Explicitly start Resonate threads. 
This happens automatically when calling `.run()` or `.rpc()`, but is generally recommended for all workers that have functions registered with Resonate."